Cocktails9/Cocktails9/TopRowView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  topRow.swift
+//  Cocktails9
+//
+//  Created by Mateja Mitic on 2.12.24..
+//
+
+import SwiftUI
+
+struct TopRowView: View {
+    @State private var searchQuery: String = ""  // For holding search text
```

Comment:
This should be used in a search field which should pop-out from bellow the NavigationBar or on a new screen.